kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -371,7 +371,7 @@ impl<'tcx> KaniAttributes<'tcx> {
                     attrs.iter().for_each(|attr| self.check_proof_attribute(kind, attr))
                 }
                 KaniAttributeKind::StubVerified => {
-                    expect_single(self.tcx, kind, &attrs);
+                    // Actual validation happens when the annotation are handled
```

Comment:
Ideally, this should be similar to `parse_stubs` so we can emit as many errors as possible, and sooner rather than later. Can you please add a test to see if that is the case today. Something like:
```rust
#[kani::proof]
#[kani::stub_verified(dummy)]
fn stub_does_not_exist() { }

#[kani::proof]
#[kani::stub_verified(dummy, dummy2)]
fn stub_invalid_args() { }
```

That said, for this PR, I think a TODO comment should suffice, since you are not introducing this limitation. Thanks!

Reply:
We already have tests for these cases in `missing_contracts_for_check.rs` and `missing_contracts_for_replace.rs`